hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -433,6 +433,208 @@
+                "sbpr": 6055,
+                "min": 0,
+                "max": 1000000000000000
+              },
+              "servicedata": {
+                "constant": 151489557,
+                "bpt": 242186,
+                "vpt": 605466012,
+                "rbh": 161,
+                "sbh": 12,
+                "gas": 1615,
+                "bpr": 242186,
+                "sbpr": 6055,
+                "min": 0,
+                "max": 1000000000000000
+              }
+            },
+            {
+              "subType": "TOKEN_FUNGIBLE_COMMON",
+              "nodedata": {
+                "constant": 7983519,
+                "bpt": 12763,
+                "vpt": 31908136,
+                "rbh": 9,
+                "sbh": 1,
+                "gas": 85,
+                "bpr": 12763,
+                "sbpr": 319,
+                "min": 0,
+                "max": 1000000000000000
+              },
+              "networkdata": {
+                "constant": 159670382,
+                "bpt": 255265,
+                "vpt": 638162730,
+                "rbh": 170,
+                "sbh": 13,
+                "gas": 1702,
+                "bpr": 255265,
+                "sbpr": 6382,
+                "min": 0,
+                "max": 1000000000000000
+              },
+              "servicedata": {
+                "constant": 159670382,
+                "bpt": 255265,
+                "vpt": 638162730,
+                "rbh": 170,
+                "sbh": 13,
+                "gas": 1702,
+                "bpr": 255265,
+                "sbpr": 6382,
+                "min": 0,
+                "max": 1000000000000000
+              }
+            },
+            {
+              "subType": "TOKEN_FUNGIBLE_COMMON_WITH_CUSTOM_FEES",
+              "nodedata": {
+                "constant": 15967037,
+                "bpt": 25527,
+                "vpt": 63816270,
+                "rbh": 17,
+                "sbh": 1,
+                "gas": 170,
+                "bpr": 25527,
+                "sbpr": 638,
+                "min": 0,
+                "max": 1000000000000000
+              },
+              "networkdata": {
+                "constant": 319340747,
+                "bpt": 510530,
+                "vpt": 1276325394,
+                "rbh": 340,
+                "sbh": 26,
+                "gas": 3404,
+                "bpr": 510530,
+                "sbpr": 12763,
+                "min": 0,
+                "max": 1000000000000000
+              },
+              "servicedata": {
+                "constant": 319340747,
+                "bpt": 510530,
+                "vpt": 1276325394,
+                "rbh": 340,
+                "sbh": 26,
+                "gas": 3404,
+                "bpr": 510530,
+                "sbpr": 12763,
+                "min": 0,
+                "max": 1000000000000000
+              }
+            },
+            {
+              "subType": "TOKEN_NON_FUNGIBLE_UNIQUE",
```

Comment:
This is missing the second place it needs to be there  for nextFeeSchedule

Reply:
Fixed 👍